modules/cdc-ext/src/main/java/org/apache/ignite/cdc/metrics/KafkaToIgniteMetrics.java

```diff
@@ -0,0 +1,214 @@
+
+    /** Count of events sent name. */
+    public static final String K2I_MSGS_SNT_CNT = "EventsSentCount";
+
+    /** Count of events sent description. */
+    public static final String K2I_MSGS_SNT_CNT_DESC = "Count of events sent to destination cluster";
+
+    /** Timestamp of last sent batch name. */
+    public static final String K2I_LAST_MSG_SNT_TIME = "LastBatchSentTime";
+
+    /** Timestamp of last sent batch description. */
+    public static final String K2I_LAST_MSG_SNT_TIME_DESC = "Timestamp of last sent batch to the destination cluster";
+
+    /** Timestamp of last received message. */
+    private AtomicLongMetric lastRcvdEvtTs;
+
+    /** Count of received events. */
+    private AtomicLongMetric evtsRcvdCnt;
+
+    /** Timestamp of last sent message. */
+    private AtomicLongMetric lastSntMsgTs;
+
+    /** Count of sent events. */
+    private AtomicLongMetric evtsSntCnt;
+
+    /** Count of received markers. */
+    private AtomicLongMetric markersCnt;
+
+    /** Standalone kernal context. */
+    private StandaloneGridKernalContext kctx;
+
+    /** CDC metrics registry. */
+    private MetricRegistryImpl mreg;
+
+    /** */
+    private final IgniteLogger log;
+
+    /** Streamer configuration. */
+    private final KafkaToIgniteCdcStreamerConfiguration streamerCfg;
+
+    /** Ignite instance name. */
+    private final String igniteInstanceName;
+
+    /** */
+    private KafkaToIgniteMetrics(
+        IgniteLogger log,
+        KafkaToIgniteCdcStreamerConfiguration streamerCfg,
+        String igniteInstanceName
+    ) throws IgniteCheckedException {
+        this.log = log;
+        this.streamerCfg = streamerCfg;
+        this.igniteInstanceName = igniteInstanceName;
+
+        initStandaloneMetricsKernal();
+        initMetrics();
+    }
+
+    /**
+     * Creates an instance of {@link KafkaToIgniteMetrics}.
+     * @param log Logger.
+     * @param streamerCfg Streamer config.
+     * @param igniteInstanceName Ignite instance name.
+     * @return {@link KafkaToIgniteMetrics} instance.
+     */
+    public static KafkaToIgniteMetrics startMetrics(
+        IgniteLogger log,
+        KafkaToIgniteCdcStreamerConfiguration streamerCfg,
+        String igniteInstanceName
+    ) {
+        try {
+            return new KafkaToIgniteMetrics(log, streamerCfg, igniteInstanceName);
+        }
+        catch (IgniteCheckedException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /** @throws IgniteCheckedException If failed. */
+    private void initStandaloneMetricsKernal() throws IgniteCheckedException {
+        kctx = new StandaloneGridKernalContext(log, null, null) {
+            @Override protected IgniteConfiguration prepareIgniteConfiguration() {
+                IgniteConfiguration cfg = super.prepareIgniteConfiguration();
+
+                cfg.setIgniteInstanceName("kafka-ignite-streamer-" + igniteInstanceName);
```

Comment:
Let's use 'CdcMain.cdcInstanceName'

Reply:
Refactored the naming for CDC metrics registries